src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -311,6 +317,48 @@ void PercussionPanelModel::playPitch(int pitch)
     delete note;
 }
 
+void PercussionPanelModel::resetLayout()
+{
+    if (m_currentPanelMode == PanelMode::Mode::EDIT_LAYOUT) {
+        finishEditing(/*discardChanges*/ true);
+    }
+
+    NoteInputState inputState = interaction()->noteInput()->state();
+    const Staff* staff = inputState.staff;
+
+    IF_ASSERT_FAILED(staff && staff->part()) {
+        return;
+    }
+
+    Instrument* inst = staff->part()->instrument(inputState.segment->tick());
```

Comment:
```suggestion
    const Instrument* inst = staff->part()->instrument(inputState.segment->tick());
```

Reply:
Sadly cannot make this const as it gets modified by `ChangeDrumset` (same goes for `finishEditing()`)